server/webapp/WEB-INF/rails/webpack/views/pages/config_repos/config_repos_widget.tsx

```diff
@@ -39,7 +39,13 @@ const yamlIcon   = require("./yaml.svg");
 const jsonIcon   = require("./json.svg");
 const groovyIcon = require("./groovy.svg");
 
-interface Operations<T> extends EditOperation<T>, DeleteOperation<T>, RefreshOperation<T> {
+export interface SearchOperation<T> {
+  initialObjects: Stream<T[] | null>;
+  searchText: Stream<string>;
+  filteredObjects: () => Stream<T[] | null>;
+}
+
+export interface Operations<T> extends EditOperation<T>, DeleteOperation<T>, RefreshOperation<T> {
```

Comment:
Should `SearchOperations` and `Operations` be moved to `page_operations.ts`.

I'm also a bit confused about the `Operations` interface. Why just Edit, Delete and Refresh? Can it be renamed to something more descriptive?